apps/web/src/pages/Profile/AccountSettings/AccountSettingsPage.tsx

```diff
@@ -0,0 +1,273 @@
+                      !featureFlags.notifications &&
+                      section.id === "notification-settings"
+                    )
+                      return null;
+
+                    return (
+                      <TableOfContents.ListItem key={section.id}>
+                        <TableOfContents.AnchorLink id={section.id}>
+                          {section.title}
+                        </TableOfContents.AnchorLink>
+                      </TableOfContents.ListItem>
+                    );
+                  })}
+                </TableOfContents.List>
+              </TableOfContents.Navigation>
+              <TableOfContents.Content>
+                <TableOfContents.Section
+                  id={sections.accountManagement.id}
+                  data-h2-padding-top="base(x3)"
+                >
+                  <TableOfContents.Heading
+                    size="h3"
+                    icon={Cog8ToothIcon}
+                    color="primary"
+                    data-h2-margin="base(0, 0, x1, 0)"
+                  >
+                    {sections.accountManagement.title}
+                  </TableOfContents.Heading>
+                  <p data-h2-margin="base(0, 0, x1, 0)">
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "This section focuses on general account management and information related to how we link to your GCKey.",
+                      id: "G6PxDn",
+                      description:
+                        "Subtitle for account management section on account settings page.",
+                    })}
+                  </p>
+                  <AccountManagement />
+                </TableOfContents.Section>
+                {featureFlags.notifications && (
+                  <TableOfContents.Section
+                    id={sections.notificationSettings.id}
+                    data-h2-padding-top="base(x3)"
+                  >
+                    <TableOfContents.Heading
+                      size="h3"
+                      icon={Cog8ToothIcon}
+                      color="secondary"
+                      data-h2-margin="base(0, 0, x1, 0)"
+                    >
+                      {sections.notificationSettings.title}
+                    </TableOfContents.Heading>
+                    <p data-h2-margin="base(0, 0, x1, 0)">
+                      {intl.formatMessage({
+                        defaultMessage:
+                          "The settings provided in this section allow you to control when you receive notifications and how they are delivered. Email notifications are delivered to the email provided on your profile, while in-app notifications are delivered to the notification pane found in the main menu next to your name.",
+                        id: "hnyf1f",
+                        description:
+                          "Subtitle for notification settings section on account settings page.",
+                      })}
```

Comment:
I'm also worried about referencing specific UI in this way:

"...the notification pane found in the main menu next to your name."

1. This isn't actually true yet because the users name does not appear in the header
2. If that ever does change, this is also becomes false

It may be easy to update this one instance but I think it sets up a problematic pattern and we may end up needed to go back and remember all the places that reference that specific UI and go back and need to change those.

Reply:
@substrae Any comments on this?

Reply:
Hmm, better?
>The settings provided in this section allow you to control the types of notifications you receive and where they are delivered. Email notifications are delivered to the contact email provided on your profile, while in-app notifications are delivered to the notification pane found in the main menu.

Reply:
Okay 👍 Fixed by [61f344c](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9971/commits/61f344c2f154f13694c407ea4eb87a99fbbfec57)